Cargo.toml

```diff
@@ -151,14 +150,14 @@ test_sst_importer = { path = "components/test_sst_importer" }
 tokio = { version = "0.2", features = ["macros", "rt-threaded", "time"] }
 zipf = "5.0.1"
 
-[replace]
+[patch.crates-io]
 # TODO: remove this when new raft-rs is published.
-"raft:0.6.0-alpha" = { git = "https://github.com/pingcap/raft-rs", default-features = false }
-"raft-proto:0.6.0-alpha" = { git = "https://github.com/pingcap/raft-rs", default-features = false }
-"protobuf:2.8.0" = { git = "https://github.com/pingcap/rust-protobuf", rev="b67d432c1b74350b38a5d96ddf885ac6c3ff46f5" }
-"protobuf-codegen:2.8.0" = { git = "https://github.com/pingcap/rust-protobuf", rev="b67d432c1b74350b38a5d96ddf885ac6c3ff46f5" }
-"prost:0.5.0" = { git = "https://github.com/danburkert/prost", rev = "1944c27c3029d01ff216e7b126d846b6cf8c7d77" }
-"protobuf-build:0.10.0" = { git = "https://github.com/tikv/protobuf-build", rev="1fe5d77689d9691b7917215f3a1d32968115cc02" }
+raft = { git = "https://github.com/pingcap/raft-rs", branch = "master" }
```

Comment:
So how is features handled in patch?

Reply:
`[patch]` uses the same feature settings with the original one.